modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -1,104 +1,75 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-request-header-identity-provider.adoc
 
 [id="identity-provider-configuring-apache-request-header_{context}"]
 = Configuring Apache authentication using request header
 
 This example configures an authentication proxy on the same host as the master.
-Having the proxy and master on the same host is merely a convenience and may not
+Having the proxy and master on the same host is merely a convenience and might not
 be suitable for your environment. For example, if you were already
-running a router
-on the master, port 443 would not be available.
+running a router on the master, port 443 would be unavailable.
 
-It is also important to note that while this reference configuration uses
-Apache's `mod_auth_gssapi`, it is by no means required and other proxies can
-easily be used if the following requirements are met:
+.Prerequisites
 
-1. Block the `X-Remote-User` header from client requests to prevent spoofing.
-2. Enforce client certificate authentication in the `RequestHeaderIdentityProvider` configuration.
-3. Require the `X-Csrf-Token` header be set for all authentication request using the challenge flow.
-4. Only the `/oauth/authorize` endpoint and its subpaths should be proxied,
+* Obtain the `mod_auth_gssapi` module from the
+link:https://access.redhat.com/solutions/392003[Optional channel].
+You must have following packages installed on your local machine:
++
+** `httpd`
+** `mod_ssl`
+** `mod_session`
+** `apr-util-openssl`
+** `mod_auth_gssapi`
+
+* Generate a CA for validating requests that submit the trusted header. Use this CA
+as the file name for `clientCA` in the identity provider configuration.
```

Comment:
"Use the file containing this CA certificate to create a configmap to be referred in the `ca` field of the `oauth/cluster` object"

My sentence is probably poorly-worded, feel free to turn it into something more florid.